load_hours.py

```diff
@@ -29,6 +29,9 @@
     notes = format_notes(entry['notes'])
 
     if 'external_reference' in entry:
+        if not entry.get('external_reference'):
+            continue
```

Comment:
@luizbraga, I'm not sure if this part of the code is needed. What do you think?

Reply:
Eita! We don't really need this for sure